TU20Bot/Handler.cs

```diff
@@ -390,7 +414,17 @@ private enum ModifyRoleOp {
                     )).FirstOrDefaultAsync();
                     
                     if (model != null) {
-                        // Remove the prompt from real life.
+                        // If this is a confirmation, then update the state to Confirmed.
+                        // This intentionally avoids checking if any tags were added on the original message so that
+                        // the author can leave the current one unindexed such that it will only appear in searches
+                         if( reaction.Emote.Name == "✅") {
+                             await eventCollection.FindOneAndUpdateAsync(
+                                Builders<EventModel>.Filter.Eq(x => x.id, model.id), 
+                                Builders<EventModel>.Update.Set(x => x.state, EventState.Confirmed)
```

Comment:
Can you do this instead in the query above so we don't need to make multiple queries? If you don't want to, use [`UpdateOneAsync`](http://mongodb.github.io/mongo-csharp-driver/2.3/apidocs/html/M_MongoDB_Driver_IMongoCollectionExtensions_UpdateOneAsync__1.htm) instead.